src/CoreFoundation/CFStream.cs

```diff
@@ -222,6 +227,11 @@ public abstract class CFStream : CFType, INativeObject, IDisposable {
 			/* CFStringRef */ IntPtr host, /* UInt32 */ int port,
 			/* CFReadStreamRef* */ out IntPtr readStream, /* CFWriteStreamRef* */ out IntPtr writeStream);
 
+		[Deprecated (PlatformName.WatchOS, 8, 0, message: "Use the new Network framework instead.")]
+		[Deprecated (PlatformName.TvOS, 15, 0, message: "Use the new Network framework instead.")]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use the new Network framework instead.")]
+		[Deprecated (PlatformName.MacCatalyst, 15, 0, message: "Use the new Network framework instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the new Network framework instead.")]
```

Comment:
same.

We already use `"Use 'Network.framework' instead."` in a lot of other places. It would be best to use the same string to reduce the metadata size of the assembly.

You could also add it to `src/ObjCRuntime/ObsoleteConstants.cs` so it's easier to avoid different string ending up inside the platform assemblies.

That would require updating (now or later) `src/Security/SecureTransport.cs` and `src/Security/SslConnection.cs`

Reply:
![Its a trap](https://media.giphy.com/media/3ornka9rAaKRA2Rkac/giphy.gif)